paket-files/fsprojects/FSharp.TypeProviders.StarterPack/src/DebugProvidedTypes.fs

```diff
@@ -0,0 +1,519 @@
+﻿// --------------------------------------------------------------------------------------
```

Comment:
Consider to gitignore the paket-files folder. This will be restored everytime you build.